IDisposableCodingExerciseWeek14/IDisposableCodingExerciseWeek14/Plane.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IDisposableCodingExerciseWeek14
+{
+    /// <summary>
+    /// Defines a plane and inherits from the Vehicle class.
+    /// </summary>
+    public class Plane : Vehicle
+    {
+        public string airplnneType { get; set; }
```

Comment:
Airplane is misspelled above. Types and comments should be spelled correct because they will be read by other developers and potentially included in documentation for customers.